content/courses/onchain-development/anchor-pdas.md

```diff
@@ -310,42 +312,45 @@ pub struct ReallocExample<'info> {
 }
 
 #[account]
+#[derive(InitSpace)]
 pub struct AccountType {
     pub data: String,
 }
+
+const DISCRIMINATOR: u8 = 8;
+const BORSH_LEN: usize = 4;
 ```
 
-Notice that `realloc` is set to `8 + 4 + instruction_data.len()`. This breaks
-down as follows:
+The `realloc` constraint from the above example can be broken down as follows:
 
-- `8` is for the account discriminator
-- `4` is for the 4 bytes of space that BORSH uses to store the length of the
-  string
+- the discriminator (8 bytes)
+- the length of the Borsh serialized data (4 bytes)
```

Comment:
The previous version was better here - this isn't the data, it's space used to store the length.

Reply:
sure, I'll revert the changes in this part.